.gitignore

```diff
@@ -85,5 +85,8 @@ MonkeyZoo/*
 # Exported monkey telemetries
 /monkey/telem_sample/
 
+# Profiling logs
+/monkey/profiler_logs/
+
```

Comment:
Won't that depend on the working directory from which you execute stuff which might change, let's say when you're running UTs? 

Probs safer to ignore all directories called `profiler_logs` 
```suggestion
# Profiling logs
profiler_logs/
```